vars/processTemplate.groovy

```diff
@@ -44,3 +51,21 @@ def applyDefaults(provided=[:], templateParams) {
   setParam('RELEASE_VERSION') { Utils.shWithOutput(this, "git rev-list --count HEAD") }
   return params
 }
+
+boolean isValidTemplate(parameters=[[:]]){
+  def pass = true
+  def mandatParamList = ["RELEASE_VERSION"]
+  def templateParam = []
+  //to convert complex hashmap in list to simple list of params.
+  parameters.each {
+    templateParam << it.name
+  }
+  //check if the mandatory params are present in template
+  mandatParamList.each {
```

Comment:
we can get rid of `pass` variable.
```
  mandatParamList.each {
    if(!templateParam.contains(it)){
      echo "Missing parameter : " + it
      return false
    }
  }
  return true
```

Also `mandatParamList` variable can be removed as only one mandatory Parameter

Reply:
I wanted to keep it there as in near future we may get another param as mandat. 
Yes now we can get rid of pass var